suite-native/device-manager/src/hooks/useDeviceManager.tsx

```diff
@@ -0,0 +1,9 @@
+import { atom, useAtom } from 'jotai';
+
+const isDeviceManagerVisibleAtom = atom(false);
+
+export const useDeviceManager = () => {
```

Comment:
Maybe `useDeviceManagerVisibility` might be more accurate?

Reply:
Ummm 🤔 I quite like just this as it **might** get extended with other properties, this hook. It's more important to have the properties with proper naming IMO but I guess both are okay